services/app-api/emailer/templates.ts

```diff
@@ -149,5 +153,73 @@ const newPackageStateEmail = (
     }
 }
 
-export { newPackageCMSEmail, newPackageStateEmail }
+type UnlockEmailData = {
+    submissionName: string
```

Comment:
Looks like our other emails have taken a whole submission here, instead of just the bits they need. Might be nice to keep that consistent but I can see it either way. This way we’re really clear what data is going into the email which is always nice. If we pass in the whole submission, then it makes it easier us to change this email to require different submision data over time.